paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentSelection.kt

```diff
@@ -257,13 +257,17 @@ internal sealed class PaymentSelection : Parcelable {
                 Configuration.UI_MODE_NIGHT_YES
         }
 
-        suspend fun loadPaymentOption(paymentOption: PaymentOption): Drawable {
```

Comment:
Removing the dependency on PaymentOption, and just requiring the necessary params.